### PR TITLE
logs: added informative logs

### PR DESCRIPTION
**Pull request by RanRegev**
during the work on 4.7.9 certification added
logs that may serve other users as well

Signed-off-by: Ran regev <ran.regev@devalore.com>

### CODE REVIEW

File: common/beerocks/bcl/source/beerocks_os_utils.cpp [hunk 1]
```diff
@@ -81,7 +81,7 @@ std::string os_utils::system_call(std::string cmd, int log_lvl, bool detached)
     // }
 
     if (system(cmd.c_str()) != 0) {
-        LOG(ERROR) << "System returned non-zero";
+        LOG(ERROR) << "System returned non-zero for the command:" << cmd.c_str();
```
Comment (tomereli) [inline common/beerocks/bcl/source/beerocks_os_utils.cpp]: If you're already changing this, why not also include the return code?
```
int ret = system(cmd.c_str());
LOG_IF(!ret, ERROR) << cmd << " failed with return code " << ret;
```

File: framework/transport/ieee1905_transport/ieee1905_transport_local_bus.cpp [hunk 2]
```diff
@@ -83,7 +83,9 @@ void Ieee1905Transport::handle_local_bus_cmdu_tx_message(CmduTxMessage &msg)
         }
 
         if (msg.metadata()->msg_type != ntohs(ch->messageType)) {
-            MAPF_WARN("CmduTxMessage messageType mismatch.");
```
Comment (tomereli) [inline framework/transport/ieee1905_transport/ieee1905_transport_local_bus.cpp:86]: OK, but this should be a separate commit.

**Review by tomereli — APPROVED** [review]
The change itself is good, but the split to commit is not. Each change should be a separate commit, please split accordingly before merging.

File: common/beerocks/bcl/source/beerocks_os_utils.cpp [hunk 3]
```diff
@@ -81,7 +81,7 @@ std::string os_utils::system_call(std::string cmd, int log_lvl, bool detached)
...
     }
 
 //if (log_lvl == 2) LOG(INFO) << "system_call ret:\n" << ret_str << "\n";
```
Comment (vitalybu) [inline common/beerocks/bcl/source/beerocks_os_utils.cpp]: Also a good oportunity to remove this comment

**Review by vitalybu — APPROVED** [review]
@tomereli has some valid comments, but since they are minor, approving anyway.
@ranregev please implement the changes before merging.

**Review by rmelotte — APPROVED** [review]
